src/bg_daemon/mac-update.sh

```diff
@@ -1,9 +1,21 @@
 #!/usr/bin/env bash
-osascript<<EOF
-tell application "Finder"
-    set desktop picture to POSIX file "$1"
-end tell
+FN=${1}-temp.jpg
+cp $1 $FN
+
+if quack -h &>/dev/null; then
```

Comment:
I'm still not completely convinced about this, it is not self-explanatory:

* Why quack -h?
* Are we testing that quack exists or that it runs successfully?

I'd rather go with an alternative that is self descriptive here:

```bash
if [-e $(which quack)] 
...
```
or
```bash
if which quack
```
For that matter.